test/check-application

```diff
@@ -1939,10 +1939,14 @@ class TestFiles(testlib.MachineCase):
         m = self.machine
 
         self.enter_files()
+        b.click("button[aria-label='Display as a list']")
 
         # Copy/paste file
         m.execute("runuser -u admin mkdir /home/admin/newdir")
         m.write('/home/admin/newfile', 'test_text\n', owner='admin:admin')
+        # "hack" test is too quick and UI doesnt reflect the chown done in m.write so it still
+        # thinks that file owner is root
+        b.wait_in_text("[data-item='newfile'] .item-owner", "admin")
```

Comment:
I don't understand this comment. The chown should generate an inotify event on the parent directory, and surely Files listens to that? Also, I don't see any "hack" here, just a proper wait condition? Is the comment obsolete?